src/gwt/src/org/rstudio/studio/client/workbench/prefs/PrefsConstants_en.properties

```
@@ -213,7 +213,8 @@ developmentNavigateLabel=Automatically navigate editor to build errors
 developmentHideLabel=Hide object files in package src directory
 developmentCleanupLabel=Cleanup output after successful R CMD check
 developmentViewLabel=View Rcheck directory after failed R CMD check
-developmentRcppLabel=Use Rcpp template when creating C++ files
+developmentCppTemplate=Use template for new C++ files

```

```suggestion
developmentCppTemplate=Use template for new C++ files:
```

For menu box labels, we typically end the label with a colon.